cloudevents-sdk-warp/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#[macro_use]
```

Comment:
Please add a basic description + example at the crate doc level (you can reuse this example https://github.com/cloudevents/sdk-rust/pull/97/files#diff-679f7de506c2dac3d52d003318062af94460810998fae27461b2aa67e447bd40)

For an example crate level doc: https://github.com/cloudevents/sdk-rust/blob/master/cloudevents-sdk-actix-web/src/lib.rs#L1